packages/editor/src/components/post-type-support-check/test/index.js

```diff
@@ -10,27 +10,26 @@ import { PostTypeSupportCheck } from '../';
 
 describe( 'PostTypeSupportCheck', () => {
 	it( 'renders its children when post type is not known', () => {
-		let postType;
-		const tree = create(
-			<PostTypeSupportCheck postType={ postType } supportKeys="title">
+		const { container } = render(
+			<PostTypeSupportCheck postType={ undefined } supportKeys="title">
```

Comment:
Is this any different than not providing a `postType` prop at all?

Reply:
No, but I just wanted to be explicit here :)